omnibus/config/software/snowflake-connector-python-py3.rb

```diff
@@ -29,5 +29,11 @@
     pip = "#{install_dir}/embedded/bin/pip3"
   end
 
-  command "#{pip} install ."
+  # We need a newer version of oscrypto than the one released to get a fix for a bug
+  # that gets triggered when having double digits on the OpenSSL version
+  # (https://github.com/wbond/oscrypto/issues/75).
+  # We can remove the oscrypto pinning once the fix becomes part of a new release
+  oscrypto_commit = "d5f3437ed24257895ae1edd9e503cfb352e635a8"
+
+  command "#{pip} install . \"oscrypto @ git+https://github.com/wbond/oscrypto.git@#{oscrypto_commit}\""
```

Comment:
I'm surprised `build_env` can be removed here, especially if we can't rely on prebuilt wheels.

But if it can, we might as well remove the `build_env` altogether since it's only used when invoking `pip`

Reply:
Oh `build_env` isn't there in 7.48 :facepalm: 

I wonder if we shouldn't backport #19747 first then, as it mentions fixing some illegal instruction errors. I'm curious to have @ofek 's opinion

Reply:
We still haven't been able to confirm that it fixes those errors, I think that's why we didn't backport that to 7.48.

Reply:
Included `build_env` based on input from @ofek

Reply:
Thanks @chouquette for the comment.